src/findcdn/cdnEngine/detectCDN/cdn_check.py

```diff
@@ -257,16 +297,20 @@ def all_checks(
         agent: str,
         verbose: bool = False,
         interactive: bool = False,
+        checks: str = "chnw",
     ) -> int:
         """Option to run everything in this library then digest."""
         # Obtain each attributes data
         self.ip(dom)
-        self.cname(dom, timeout)
-        self.https_lookup(dom, timeout, agent, interactive, verbose)
-        self.whois(dom, interactive, verbose)
+        if 'c' in checks:
```

Comment:
I really like this concept. I have thought running through all checks is kind of annoying. Giving options is nice.